README.md

```diff
@@ -51,6 +51,14 @@ It will automatically start tracing.
 **This tool may also (temporarily) corrupt the state of your Xbox.**
 If this tool does not work, please retry a couple of times.
 
+### Contributing
+
+This project uses [Black](https://pypi.org/project/black/) for automatic formatting.
+You can use the pre-commit in the `githooks` directory to reformat any changes on commit
```

Comment:
I'd rather not have githooks as part of the repository (personal preference, and I found them annoying in projects that used them.

Reply:
Not a fan, but other maintainers should decide